src/plugins/common/format/lib/format-plugin.js

```diff
@@ -726,7 +726,7 @@ define('format/format-plugin', [
 		 *
 		 * @todo new buttons needed for 'code'
 		 */
-		availableButtons: Object.keys(DEFAULT_CONFIG).concat(REMOVE_FORMAT_ID),
```

Comment:
Check if another plugin modifies this as well

Reply:
No other instances found.